.gitignore

```diff
@@ -0,0 +1,2 @@
+tags
```

Comment:
make sure you also look at this:
```
def something(here):
    print('something')
```

also check out `this variable`.

screenshot:

![image](https://cloud.githubusercontent.com/assets/704593/22447125/65ea284e-e705-11e6-9201-e3619aec5b38.png)